app/main.py

```diff
@@ -1 +1,40 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive: list = []
+
+    def __init__(self, name: str, health: int = 100) -> None:
+        self.name = name
+        self.health = health
+        self.hidden = False
+        Animal.alive.append(self)
+
+    def is_alive(self) -> bool:
+        return self.health > 0
+
+    @staticmethod
+    def remove_dead(animal: Herbivore) -> None:
+        if not animal.is_alive():
+            Animal.alive.remove(animal)
```

Comment:
Code Efficiency: Use the class name to get access to the class attributes. It's better to use `Animal.alive.remove(animal)` instead of `self.alive.remove(animal)`. This way, it's clear that we're accessing a class attribute, not an instance attribute.